daemon/src/routes_taker.rs

```diff
@@ -159,27 +161,55 @@ pub async fn post_cfd_action(
 }
 
 #[derive(Debug, Clone, Deserialize)]
-pub struct WalletReinitialiseRequest {
-    pub seed_words: String,
+pub struct WalletRestoreRequest {
+    pub mnemonic: String,
```

Comment:
`Mnemonic` implements `Deserialize`: https://docs.rs/bip39/1.0.1/bip39/struct.Mnemonic.html#impl-Deserialize%3C%27de%3E

Can we use that here?